restli-server/src/main/java/com/linkedin/restli/internal/server/response/BatchFinderResponseBuilder.java

```diff
@@ -195,24 +195,24 @@ private AnyRecord buildMetaData(CollectionResult<RecordTemplate, RecordTemplate>
     return null;
   }
 
-  private String getBatchParamterName(RoutingResult routingResult)
+  private String getBatchParameterName(RoutingResult routingResult)
   {
     int batchFinderCriteriaIndex = routingResult.getResourceMethod().getBatchFinderCriteriaParamIndex();
     return routingResult.getResourceMethod().getParameters().get(batchFinderCriteriaIndex).getName();
   }
 
   private DataList getCriteriaParameters(RoutingResult routingResult)
   {
-    String batchParamName = getBatchParamterName(routingResult);
-    return(DataList)routingResult.getContext().getStructuredParameter(batchParamName);
+    String batchParameterName = getBatchParameterName(routingResult);
+    return (DataList) routingResult.getContext().getStructuredParameter(batchParameterName);
   }
 
-  private URI buildCriteriaURI(ResourceContextImpl resourceContext, RecordTemplate criteria, String queryParam, URI uri)
+  static URI buildCriteriaURI(ResourceContextImpl resourceContext, RecordTemplate criteria, String batchParameterName, URI uri)
```

Comment:
Is this for testing purpose only? If yes, we should add a comment "package-private for testing purpose".

Reply:
Sorry just saw this, will do in my next PR.

Reply:
This is for testing only right now